mros2_msgs/srv/NodeMode.srv

```diff
@@ -0,0 +1,3 @@
+---
+string node_name
+string mode_name
```

Comment:
Aren't `node_name` and `mode_name` supposed to be part of the request? Like this they are part of the response